packages/flutter_test/lib/src/binding.dart

```diff
@@ -395,6 +395,15 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     assert(debugAssertNoTransientCallbacks(
       'An animation is still running even after the widget tree was disposed.'
     ));
+    assert(debugAllRenderVarsUnset(
```

Comment:
looks like the convention is debugAssertFoo... rather than just debugFoo, but I'll leave that up to you.

LGTM.